tests/sqlparser_common.rs

```diff
@@ -17905,3 +17905,22 @@ fn test_parse_set_session_authorization() {
         }))
     );
 }
+
+#[test]
+fn parse_select_distinct_parenthesized_wildcard() {
```

Comment:
```suggestion
fn parse_select_parenthesized_wildcard() {
```
thinking since the syntax isn't implemented solely on the distinct keyword?